docs/api/advanced/snapshot.mdx

```diff
@@ -28,9 +28,9 @@ const snap3 = snapshot(store)
 console.log(snap1 === snap3) // false, should re-render
 ```
 
-## Copy on Write
+## <a name="copyonwrite"></a> Copy on Write
```

Comment:
I think there's automatically generated link in github browser. (I'm not sure how it does with `website`.

```suggestion
## Copy on Write
```

Reply:
Got it - changed it to the correct path and removed the anchor